authors/apps/authentication/tests/test_social_auth.py

```diff
@@ -1,35 +1,38 @@
 from django.test import TestCase
+from django.urls import reverse
 from rest_framework import test, status
 
 
 class SocialAuthTest(TestCase):
+    def setUp(self):
+        self.client = test.APIClient()
 
     # FACEBOOK
-    def test_user_signup_success(self):
-        pass
-
-    def test_user_signup_invalid_fields(self):
-        pass
-
-    def test_empty_fields(self):
-        pass
-
-    # TWITTER
-    def test_user_signup_success(self):
-        pass
-
-    def test_user_signup_invalid_fields(self):
-        pass
-
-    def test_empty_fields(self):
-        pass
+    def test_user_signup_page_redirects_to_facebook(self):
+        """
+            Ensures that the redirects are found, basically tests
+            that the configurations are functional
+        """
+        res = self.client.get(
+            reverse('social:begin', kwargs={'backend': 'facebook'}))
+        self.assertEqual(res.status_code, status.HTTP_302_FOUND) 
+
+    # Github
```

Comment:
The story stated that the social login platforms to be used are Facebook, Twitter and GitHub. Is there a reason for including GitHub as well?

Reply:
Yes. Indeed it is Twitter on the PT board, however, I could not get it set up on localhost because Twitter does not support localhost connections. I thus implemented Github instead. I will, however, set up configurations for twitter for when we go live, only one downside, there is no quality-assured since it is not tested locally.